### PR TITLE
Add support for Resource Accounting via systemd

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
Depends on https://github.com/camptocamp/puppet-systemd/pull/66

**Comment by bastelfreak** [comment]
Depends on https://github.com/camptocamp/puppet-systemd/pull/67

File: manifests/init.pp [hunk 1]
```diff
@@ -33,15 +33,17 @@
...
+  Boolean                         $manage_resolved,
+  Enum['stopped','running']       $resolved_ensure,
+  Boolean                         $manage_networkd,
+  Enum['stopped','running']       $networkd_ensure,
+  Boolean                         $manage_timesyncd,
+  Enum['stopped','running']       $timesyncd_ensure,
+  Optional[Variant[Array,String]] $ntp_server,
+  Optional[Variant[Array,String]] $fallback_ntp_server,
+  Boolean                         $manage_accounting,
+  Hash                            $accounting,
```
Comment (ekohl) [inline manifests/init.pp]: Could this be a  ```Hash[String, String]```?

File: spec/classes/init_spec.rb [hunk 2]
```diff
@@ -67,6 +67,28 @@
           it { is_expected.to compile.with_all_deps }
           it { is_expected.to contain_systemd__service_limits('openstack-nova-compute.service').with_limits({'LimitNOFILE' => 32768}) }
         end
+
+        context 'when managing Accounting options' do
```
Comment (ekohl) [inline spec/classes/init_spec.rb]: I also like adding ```it { is_expected.to compile.with_all_deps }``` to my tests

Comment (bastelfreak) [reply spec/classes/init_spec.rb]: updated it

File: manifests/init.pp (hunk 1 above)
Comment (bastelfreak) [reply manifests/init.pp]: fixed

**Comment by raphink** [comment]
#67 was merged, so  you can refactor this one.

**Comment by bastelfreak** [comment]
@raphink rebased it.

**Comment by ekohl** [comment]
Could you also fix the typo in the title of this PR? :)

**Comment by bastelfreak** [comment]
I think I fixed it :D

**Comment by sirinek** [comment]
With this enhancement, the module is working great for me. Any thoughts about cutting a new release or moving it to voxpupuli?

**Pull request merged**